src/config.rs

```diff
@@ -8,15 +9,19 @@ const ENV_VAR_NAME: &str = "INDEXER_CONFIG_PATH";
 
 pub const SERVER_ADDR: &str = "127.0.0.1";
 pub const SERVER_PORT: u16 = 30303;
+pub const SERVER_PORT_STR: &str = "30303";
 
 pub const TENDERMINT_ADDR: &str = "http://127.0.0.1";
 pub const INDEXER_PORT: u16 = 26657;
+pub const INDEXER_PORT_STR: &str = "26657";
 
 pub const JAEGER_HOST: &str = "localhost";
 pub const JAEGER_PORT: u16 = 6831;
+pub const JAEGER_PORT_STR: &str = "6831";
 
 pub const PROMETHEUS_HOST: &str = "localhost";
 pub const PROMETHEUS_PORT: u16 = 9000;
+pub const PROMETHEUS_PORT_STR: &str = "9000";
```

Comment:
I would rather avoid the redundancy. There is no way to set the default value as `u16` ?

Reply:
maybe better now